en/lessons/advanced/nerves.md

```diff
@@ -0,0 +1,263 @@
+---
+version: 1.0.0
+title: Nerves
+---
+
+## Introduction and requirements
+
+We will be talking about Nerves in this lesson. Nerves project is a framework for using Elixir in embedded software development. As their site says, `Craft and deploy bulletproof embedded software in Elixir`. Due to the topic, this lesson will be a bit different from other Elixir School lessons. Nerves is an advanced topic and require some advanced system setup and additional hardware, as well as understanding what you are doing. 
+
+Today we are going to be exploring this land of embedded development. So, if you have hardware from their list of [supported targets](https://hexdocs.pm/nerves/targets.html), card reader with a memory card supported by the hardware of your choice + wired networking connection to access this device by the network, we are ready to start. 
+
+However, we suggest you have a Raspberry Pi, due to it having controllable LED onboard. We also advise you have a screen connected to your hardware of choice - this will simplify debugging a lot due to having IEx.
+
+## Setup
+
+Nerves project itself has an excellent [Getting started guide](https://hexdocs.pm/nerves/getting-started.html), but let's try approaching from a "fewer words, more code" approach.
+
+Firstly, you need an environment set up. You can find the guide in the [Installation](https://hexdocs.pm/nerves/installation.html) part of Nerves wiki. You may want to skip it, however, please make sure that you have the same version of both OTP and Elixir they are mentioning. This check could save you sometime later on. At the moment of writing, any OTP 21 Elixir should work.
```

Comment:
some time

any OTP 21 Elixir (version | subversion) should work

Reply:
Not sure what you mean in the second part, rephrased.